Tests/ApolloCacheDependentTests/FetchQueryTests.swift

```diff
@@ -473,5 +473,8 @@ class FetchQueryTests: XCTestCase {
     }
     self.wait(for: [expectation], timeout: 10)
     
+    for watcher in watchers {
+      watcher.cancel()
+    }
```

Comment:
I have a sneaking suspicion that the watchers are getting overcalled because they're not necessarily getting killed off by ARC in time to avoid calls from other tests.

Reply:
[Nope - still failing occasionally](https://circleci.com/gh/apollographql/apollo-ios/4380)

Reply:
not sure if its the case here, but we had to add to wrap to `autoreleasepool { ... }` to workaround ARC

Reply:
what'd you need to add that to?